### DISCUSSION TITLE
Maximum retries exceeded - Help 🙏

### DISCUSSION THREAD

**Discussion by GermanBartoli** [anuraghazra/github-readme-stats#2403] [Q&A] [answered]
Something went wrong! file an issue at https://tiny.one/readme-stats
Maximum retries exceeded Please add an env variable called PAT_1 with your github token in vercel

``` html
<a
   href="https://github.com/bartoligerman497"
   target="_blank"
   rel="noreferrer">
    <img
        src="https://github-readme-stats.vercel.app/api/top-langs/?username=bartoligerman497&layout=compact&langs_count=10&hide=Rich+Text+Format&hide_border=true&title_color=0366d6&text_color=777777&icon_color=959da5&bg_color=ffffff00"
        width="70%"
        height=""
        alt="Lenguages"
    />
</a>
```

![image](https://user-images.githubusercontent.com/53313625/212321238-041139ab-b21f-4469-8bc7-a1d1866aa7d8.png)

![image](https://user-images.githubusercontent.com/53313625/212320639-d4219e5b-80c4-4a4d-9779-eeff369d7bf7.png)

**Comment by pedrolbacelar** [comment]
Same problem here, just copied and pasted (changing to my username) and got the same problem

**Reply by pedrolbacelar** [reply]
Getting the same error when trying to add `hide`: 
```
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=pedrolbacelar&count_private=true&theme=tokyonight&hide=stars,commits,prs)](https://github.com/anuraghazra/github-readme-stats)
```

**Comment by pencilnav** [comment, marked as answer]
Use another URL. you can find them here: https://github.com/anuraghazra/github-readme-stats/issues/2415
